### PR TITLE
egar-public-site-ui - settings for ECR

### PR DESCRIPTION
**Pull request by inaki-guadalajara**
**What changes does this PR bring?**

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [x] Have you built the code locally and confirmed its working?
- [x] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

**Comment by inaki-guadalajara** [comment]
Settings for ECR

File: .drone.yml [hunk 1]
```diff
@@ -442,21 +444,24 @@ steps:
...
-      DOCKER_PASSWORD:
-        from_secret: DOCKER_PASSWORD
-      DOCKER_REGISTRY_URL:
-        from_secret: DOCKER_REGISTRY_URL
-      DOCKER_USERNAME:
-        from_secret: DOCKER_USERNAME
+      AWS_ACCESS_KEY_ID:
+        from_secret: DEV_EGAR_PUBLIC_SITE_UI_AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY:
+        from_secret: DEV_EGAR_PUBLIC_SITE_UI_AWS_SECRET_ACCESS_KEY
```
Comment (ame-test) [inline .drone.yml:450]: Question: Is the cron job possible to test or not?
* ideally should be tested, but can be left.

**Review by ame-test — APPROVED** [review]
This looks good

File: .drone.yml (hunk 1 above)
Comment (inaki-guadalajara) [reply .drone.yml:364]: 1. - Tested successfully:
https://drone-gh.acp.homeoffice.gov.uk/UKHomeOffice/egar-public-site-ui/4241
2.- Please re-review the PR as I have added a few scripts to simplify maintainability

File: .drone.yml [hunk 2]
```diff
@@ -364,20 +294,8 @@ steps:
...
-      - export DOCKER_REGISTRY_URL=$${DOCKER_REGISTRY_URL}
-      - export KUBE_ARTIFACTORY_APP_SECRET=$${PRODUCTION_KUBE_ARTIFACTORY_APP_SECRET}
-      - export KUBE_SERVER=$${PRODUCTION_KUBE_SERVER}
-      - export KUBE_TOKEN=$${PRODUCTION_KUBE_TOKEN}
-      - export PUBLIC_SITE_EXTERNAL_URL=$${PRODUCTION_BASE_URL_SERVICE}
-      - source ./ops/common_lib
-      - source ./ops/prod_values
-      - kd --timeout=6m --file ops/kube/artifactory-secret.yml
-      - kd --timeout=6m --file ops/kube/public-site-network-policy.yml
-      - kd --timeout=6m --file ops/kube/public-site-deployment.yml
```
Comment (ame-test) [inline .drone.yml:377]: Question: Production only deploys an `external` ingress.
* `deploy.sh` gives both an external and internal deployment.
* Would this be an issue?

**Review by ame-test — CHANGES_REQUESTED** [review]
Questions should be answered before approving.

**Comment by inaki-guadalajara** [comment]
All issues addressed. Not sure if any other changes are needed. @ame-test

**Review by ame-test — APPROVED** [review]
It is good.

**Pull request merged**